scipy/optimize/_differentialevolution.py

```diff
@@ -181,6 +189,54 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     >>> result.x, result.fun
     (array([ 0.,  0.]), 4.4408920985006262e-16)
 
+
+    Next example demonstrates the parallelization capabilities.
+
+    >>>
+    from scipy.optimize import rosen
+    from scipy.optimize import differential_evolution as de
+    from scipy.misc import SPool
+    from scipy.misc import PPool
+    import time
+
+    def objfunc(params):
+        for it in xrange(1000000):
```

Comment:
I also think that the example is misleading.  It's not clear that the parallel version will converge in a smaller amount of generations/function evaluations than the serial version.  The example indicates that's it's always better to go parallel. 
I can think of two contrasting examples where it might be better to go serial, and one where it might be worthwhile to go parallel.
Serial Example:
The objective function is cheap (fast) to calculate.  Because of the way that the population is evolved a minimum may be reached in a smaller number of generations.
Parallel example:
The objective function is expensive to calculate.  Whilst the aggressive approach might use a smaller number of generations to reach a minimum the quasi-aggressive approach is quicker because you can run through generations more quickly.